### ISSUE TITLE
Doc hardware requirement

### ISSUE THREAD

**Issue by frodrigo** [CanalTP/mimirsbrunn#348]
Can you add to the doc the hardware requirement for different areas (France, World) in terms of RAM and disk.

**Comment by riendegris** [comment]
Hello Frederic,

I agree this information is missing from the documentation, and we will add it.

The documentation certainly need some TLC, and I will work on it. Maybe you can share your experience of writing documentation with OSM? Could it be possible to get in touch w/ you to see how to improve coordination?

**Comment by frodrigo** [comment]
For now I am just evaluating Mimirsbrunn. Guess what is worst point of the project ?... the doc.

**Comment by riendegris** [comment]
I agree the documentation needs work, a lot of work. I don't know if it the worst though. It certainly doesn't help onboarding people like you who might be interested. If you're willing to help me with that, I'd love to improve that aspect of the project.
